src/components/tooltip/tooltip.tsx

```diff
@@ -17,6 +17,7 @@ interface TooltipOwnProps {
   maxWidth?: string;
   label: string;
   strictPosition?: boolean;
+  className?: string;
```

Comment:
this is too generic name, its role is not clear

Reply:
the role is className, we do the same in other places where we want to pass calssNames as props.